packetery/src/Packetery/Module/Carrier/CountryListingPage.php

```diff
@@ -83,7 +83,13 @@ public function render(): void {
 			delete_transient( 'packetery_run_update_carriers' );
 		}
 
-		$carriersUpdateParams['link']       = $this->httpRequest->getUrl()->withQueryParameter( 'update_carriers', '1' )->getRelativeUrl();
+		$carriersUpdateParams['link']       = add_query_arg(
+			[
+				'page'            => 'packeta-country',
```

Comment:
packeta-country se používá na několika místech.  Nezasloužilo by si to už nějakou konstantu např. ?